### ISSUE TITLE
Is the project still maintained? 

### ISSUE THREAD

**Issue by hzitoun** [raghakot/keras-vis#221]
**Docs 404**

Conv filter visualization https://raghakot.github.io/keras-vis/visualizations/conv_filters **doesn't work**

Dense layer visualization https://raghakot.github.io/keras-vis/visualizations/dense **doesn't work**

**Comment by soumitra9** [comment]
Yeah, most of the module doesn't exist.  I can't believe I wasted a day on this.

**Comment by bersbersbers** [comment]
I think it is fair to conclude that  this project is not maintained any longer. Missing functionality is unlikely to be added. Also, the more TensorFlow evolves, the more incompatibilities will be introduced, and the less likely a simple change from `keras` to `tf.keras` will make it work. People should consider using https://github.com/keisen/tf-keras-vis/ instead, or live with having to use an old, unsupported version of TensorFlow and Keras instead.
